### DISCUSSION TITLE
How to get notifications for latest release

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#90] [Ideas]
I like to always have the latest release of Jitendex (plus I can see right away if there are any changes)
Previously you could "watch" the repo and get notifications for new releases. Since the downloads are now hosted directly on the site, there's no way to get notifications. Is there any sort of solution for this?

**Comment by stephenmk** [comment]
The [static-site generator](https://getpelican.com/) that I'm using includes the ability to set up an RSS feed. I've been meaning to get around to setting it up sooner or later. I'd also like to include a summary of data updates that are made between each release (broken into sections with new entries, entries with updated metadata, entries with updated glosses, etc.).

**Comment by stephenmk** [comment]
Also, you can just watch for new releases in [this repo](https://github.com/stephenmk/stephenmk.github.io) since that's where I'm uploading them now. They won't include any release notes, though.

**Comment by parfait8566** [comment]
> Also, you can just watch for new releases in [this repo](https://github.com/stephenmk/stephenmk.github.io) since that's where I'm uploading them now. They won't include any release notes, though.

Would it be possible to start adding release notes? No pressure if it'd be too much effort or you're just not interested

**Reply by stephenmk** [reply]
I am posting release notes in the main Jitendex repo as well whenever there's a new numbered version (4.5, 4.6, 5.0, etc).
